lstchain/irf/hdu_table.py

```diff
@@ -309,13 +307,16 @@ def create_event_list(
 
     src_sky_pos = SkyCoord(
         alt=reco_alt, az=reco_az, frame=AltAz(obstime=time_utc, location=location)
-    ).transform_to(frame="icrs")
+    )
     tel_pnt_sky_pos = SkyCoord(
-        alt=pointing_alt.mean(),
-        az=pointing_az.mean(),
-        frame=AltAz(obstime=mean_time, location=location),
+        alt=pointing_alt[0],
+        az=pointing_az[0],
+        frame=AltAz(obstime=time_utc[0], location=location),
     ).transform_to(frame="icrs")
 
+    with erfa_astrom.set(ErfaAstromInterpolator(30 * u.s)):
+        src_sky_pos_icrs = src_sky_pos.transform_to(frame="icrs")
```

Comment:
If src pos is already in icrs, then why transform it here again?

Reply:
The positions being transformed are from the events, while, the `source_pos` I use is for metadata and to compare with the pointing positions to evaluate the observation mode or any 'mis-pointing'